helm/prometheus-rules/templates/kaas/tenet/alerting-rules/capi-kubeadmconfig.rules.yml

```diff
@@ -10,7 +10,16 @@ spec:
     - name: capi-kubeadmconfig
       rules:
         - alert: KubeadmConfigNotReady
-          expr: capi_kubeadmconfig_status_condition{type="Ready", status="False"} > 0
+          expr: |-
+            (
+              app_operator_app_info{status="not-installed", catalog=~"giantswarm|cluster|default", team!~"^$|noteam"}
```

Comment:
What does that have to do with the app operator?

Reply:
welp, what did I do!?? I screwed the query with a broken copy/paste, and there were no unit tests to catch it! :scream: 
Thanks for catching it!

Reply:
I really wonder how I did it, because my Grafana Explore panel was actually using the right query :thinking: 
![image](https://github.com/user-attachments/assets/231b79d2-4b88-4d8a-873f-87e9dc2fc990)

Anyway, hopefully I've now pushed the proper one :crossed_fingers: